library/core/src/ptr/non_null.rs

```diff
@@ -1390,10 +1376,10 @@ impl<T: ?Sized> NonNull<T> {
     /// ```
     ///
     /// [tracking issue]: https://github.com/rust-lang/rust/issues/104203
+    #[inline]
+    #[must_use]
```

Comment:
Do we have a preferred attribute order? I'm never sure which order people prefer, if anyone even cares. If there is preference, I'd prefer to have a tool to make it happen.

Reply:
Whenever I touch functions I sort them from smallest to largest (exception being `rustc_stable` always before `rustc_const_stable`), which does seem a bit silly but looks nice IMO. Looking through libs, I think this is somewhat common?

Didn't actually mean to change these here since they're not related stabilization, but it doesn't hurt anything.